be/src/exec/spill/input_stream.h

```diff
@@ -27,12 +28,15 @@ namespace starrocks::spill {
 
 // InputStream is used in restore phase to represent an input stream of a restore task.
 // InputStream reads multiple Blocks and returns the deserialized Chunks.
-class InputStream {
+class SpillInputStream;
+using InputStreamPtr = std::shared_ptr<SpillInputStream>;
+
+class SpillInputStream {
```

Comment:
Spill prefix is not necessary, we have put it into a separate namespace called spill

Reply:
We already have class InputStream, which may come with unnecessary trouble in the future

Reply:
> We already have class InputStream, which may come with unnecessary trouble in the future

I added the spill namespace just to make a distinction. When using it, you need to explicitly specify spill::InputStream, and there is no need to add an extra prefix.